### PR TITLE
aquavanjaram I8II support

### PR DESCRIPTION
**Pull request by cmingch**

### CODE REVIEW

**Review by KKyang — APPROVED** [review]

File: clients/gtest/matmul_gtest.yaml [hunk 1]
```diff
@@ -916,6 +916,21 @@ Tests:
   unit_check: 1
   gpu_arch: '90a'
 
+- name: matmul_gemm_i8_94x
+  category: pre_checkin
+  function:
+    matmul: *interger8_precisions
```
Comment (jichangjichang) [inline clients/gtest/matmul_gtest.yaml:922]: interger8 -> integer8

File: clients/include/hipblaslt_common.yaml [hunk 2]
```diff
@@ -93,7 +93,7 @@ Real precisions xf32: &real_precisions_intermeddiate
 
 Real precisions i8: &interger_precisions_i8
```
Comment (jichangjichang) [inline clients/include/hipblaslt_common.yaml:94]: interger-> integer

File: clients/gtest/matmul_gtest.yaml [hunk 3]
```diff
@@ -916,6 +916,21 @@ Tests:
...
+  matrix_size:
+    - { M:  128,  N:  128,  K:  128  }
+    - { M:  131,  N:  131,  K:  131  }
+    - { M:  1024, N:  1024, K:  1024 }
+    - { M:  1031, N:  1031, K:  1031 }
+  transA_transB: *transA_transB_range
+  alpha: 1
+  beta: [ 0, 2 ]
```
Comment (jichangjichang) [inline clients/gtest/matmul_gtest.yaml:930]: add supported epilogue function into test cases

File: library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/aquavanjaram/gfx942/Equality/aquavanjaram_Cijk_Ailk_Bjlk_I8II_BH_AI_SAV.yaml [hunk 4]
```diff
@@ -0,0 +1,1354 @@
...
+    GlobalReadVectorWidthA: 4
+    GlobalReadVectorWidthB: 8
+    GlobalSplitU: 1
+    GlobalSplitUAlgorithm: MultipleBuffer
+    GlobalWriteVectorWidth: 1
+    GroupLoadStore: false
+    GuaranteeNoPartialA: false
+    GuaranteeNoPartialB: false
+    GuaranteeNoPartialMetadata: true
+    ISA: [9, 4, 0]
```
Comment (jichangjichang) [inline library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/aquavanjaram/gfx942/Equality/aquavanjaram_Cijk_Ailk_Bjlk_I8II_BH_AI_SAV.yaml:110]: 942

File: clients/include/hipblaslt_common.yaml (hunk 2 above)
Comment (cmingch) [reply clients/include/hipblaslt_common.yaml]: done

File: clients/gtest/matmul_gtest.yaml (hunk 3 above)
Comment (cmingch) [reply clients/gtest/matmul_gtest.yaml:930]: done

File: library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/aquavanjaram/gfx942/Equality/aquavanjaram_Cijk_Ailk_Bjlk_I8II_BH_AI_SAV.yaml (hunk 4 above)
Comment (cmingch) [reply library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/aquavanjaram/gfx942/Equality/aquavanjaram_Cijk_Ailk_Bjlk_I8II_BH_AI_SAV.yaml]: done

File: clients/gtest/matmul_gtest.yaml (hunk 1 above)
Comment (cmingch) [reply clients/gtest/matmul_gtest.yaml]: done

**Review by jichangjichang — APPROVED** [review]
lgtm

**Pull request merged**